tests/labhub_test.py

```diff
@@ -261,6 +261,10 @@ def test_mark_cmd(self):
         mock_gitlab_mr.labels = ['process/wip']
         testbot.assertCommand(cmd_github.format('pending', 'coala', 'a', '23'),
                               'marked pending review')
+        testbot.assertCommand(cmd.format('pending-review', 'coala', 'a', '23'),
```

Comment:
@LifetimeAlone Change `cmd.format` --> `cmd_github.format` in this line and next line.